File: app/src/main/java/com/vibhorsrv/cameraids/util/AppUtil.java
package com.vibhorsrv.cameraids.util;

import android.content.Context;
import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;

public class AppUtil {

    public static String getVersionName(Context context) {
        try {
            PackageInfo packageInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
            return packageInfo.versionName;
        } catch (PackageManager.NameNotFoundException e) {
            e.printStackTrace();
        }
        return "";

    }
}

File: app/src/main/java/com/vibhorsrv/cameraids/util/AppUtil.kt
package com.vibhorsrv.cameraids.util

import android.content.Context
import android.content.pm.PackageManager

object AppUtil {
    fun getVersionName(context: Context): String {
        try {
            val packageInfo = context.packageManager.getPackageInfo(context.packageName, 0)
            return packageInfo.versionName
        } catch (e: PackageManager.NameNotFoundException) {
            e.printStackTrace()
        }
        return ""
    }
}
